### DISCUSSION TITLE
An absolute path to '/...' is included in the 'nav' configuration, which presumably points to an external

### DISCUSSION THREAD

**Discussion by viceice** [squidfunk/mkdocs-material#6318] [Q&A] [answered]
Hi, we're getting this warning:

```
An absolute path to '/...' is included in the 'nav' configuration, which presumably points to an external
```
- https://github.com/renovatebot/renovatebot.github.io/pull/305

We're not sure where it's coming from. We're using the `awesome-pages` plugin.
Any hint how to solve that?

**Comment by squidfunk** [comment]
Thanks for asking. I think it's a good idea to ask the maintainers of the [`awesome-pages` plugin](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/).

**Reply by viceice** [reply]
okay, will do. thanks. 🤗

**Reply by viceice** [reply]
- https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/issues/92

**Reply by viceice** [reply, marked as answer]
Removing `nav` from `mkdoc.yml` solved the error.
